app/src/main/res/layout/include_new_browser_tab.xml

```diff
@@ -55,9 +55,10 @@
 
                 <ImageView
                     android:id="@+id/ddgLogo"
-                    android:layout_width="wrap_content"
+                    android:layout_width="140dp"
```

Comment:
can you make this a dimension? we will use this logo in other places and having the dimension will help to keep it consistent.

Reply:
I thought about it but when checking `design-system-dimensions.xml` we just have dimensions related to components or "Legacy". 
I'll add it under a new "section comment" for New Tab Page 👍